test/common/quota.go

```diff
@@ -27,20 +27,23 @@ import (
 )
 
 const (
-	prometheusRule1           = "marin3r-api-usage-alert-level1"
+	prometheusRule1           = "api-usage-alert-level1"
 	prometheusRule1Desc       = "per minute over 4 hours"
-	prometheusRule2           = "marin3r-api-usage-alert-level2"
+	prometheusRule2           = "api-usage-alert-level2"
 	prometheusRule2Desc       = "per minute over 2 hours"
-	prometheusRule3           = "marin3r-api-usage-alert-level3"
+	prometheusRule3           = "api-usage-alert-level3"
 	prometheusRule3Desc       = "per minute over 30 minutes"
 	higherQuotaParam          = "200"
 	lowerQuotaParam           = "1"
 	higherQuotaName           = "20 Million"
+	higherQuotaParamMT        = "10"
```

Comment:
I think 100 is the value we use to set the quota to 10million
```suggestion
	higherQuotaParamMT        = "100"
```

Reply:
adressed. Changed the name of the quota to 1 Million. Nicely spotted!